### ISSUE TITLE
Goofy ahh User profile

### ISSUE THREAD

**Issue by foursplaysroblox** [moyaiu/OldCord#35]
User profile flyout is broken
![image](https://user-images.githubusercontent.com/82078139/209420285-32f575fe-164b-4701-bf9a-ea71e24449a1.png)

**Comment by DaMoyai** [comment]
what a funny and goofy a/b rollout that i dont have!

**Comment by foursplaysroblox** [comment]
![QuandaleQuandaleDingleGIF](https://user-images.githubusercontent.com/82078139/209454554-0b9a223a-5d03-4903-ada6-381f85093a12.gif)

**Comment by DaMoyai** [comment]
https://user-images.githubusercontent.com/51537072/209461405-d192603e-bbb9-4e8b-83d8-acd19f7de64d.mp4

**Comment by foursplaysroblox** [comment]
quandile dingle here

**Comment by DaMoyai** [comment]
What's up guys! It's Quandale Dingle here! (RUUEHEHEHEHEHEEHE) I have been arrested for multiple crimes (AHHHHHHHHHHHHH) including: Battery on a police officer (WHAT), Grand theft, Declaring war on Italy, and public indecency (RUHEHEHEEHEHEHEHEHEHEHE X2 speed). I will be escaping prison on, MARCH 28TH! After that.... I WILL TAKE OVER THE WORLD

Greetings! Quandale Dingle here. My cousin Henry Dinglenut got arrested for putting TNT in a daycare center. (WHAT THE FUCK??) I put a camera in Joe Biden's bathroom and watched him take a poop. (WHAT????) My Asian brother, Quanliling Dingle put illegal substances in my ramen and I died.

Hey, Quandale Dingle here. I just escaped prison and staying at Juandale Pringle's house. As I was running away from cops, I fell and scraped some of my foreskin off. A guy named Garfield Jenson bit me over in the shower while I was in prison. My baby momma Shiniqua Inderson told me to pay child support so I gave my baby to a creepy old guy,

Hey fellas it's Quandele Dingle here I put perks on Vladimir Putin's drink And he went to bed for a really long time I trapped my autistic son's hand in an air fryer (herrrrr) I dumped boiling water on a prison guard's head My goofy ah friend, Jamarious Quandele Dingle tried to eat my but during Ramadhan
